### DISCUSSION TITLE
How to get the chromaticity coordinates of an illuminant?

### DISCUSSION THREAD

**Discussion by disidrosi** [colour-science/colour#1243] [Q&A] [answered]
I am trying to calculate the CIE x and y values for 2° or 10° of a given illuminant (for example D65 or A).

I am sure there is some easy way to do this. I have found `colour.plotting.diagrams.plot_sds_in_chromaticity_diagram`, which seems to already implement the conversion, but I can't seem to figure out how to do just the conversion alone.

I also tried:

```python
D65_illuminant = color.SDS_ILLUMINANTS["D65"]
two_degree_cmfs = color.MSDS_CMFS["CIE 1931 2 Degree Standard Observer"]
D65_XYZ = color.sd_to_XYZ(D65_illuminant, two_degree_cmfs, D65_illuminant)
D65_xy = color.XYZ_to_xy(D65_XYZ)
```

which returns `array([ 0.29283577,  0.32305116])`. Am I missing something here?

**Comment by disidrosi** [comment, marked as answer]
Nevermind, shortly after posting my question I found the solution by using `colour.CCS_ILLUMINANTS`. For example:

```python
color.CCS_ILLUMINANTS['CIE 1931 2 Degree Standard Observer']['D65']
```

which returns `array([ 0.3127,  0.329 ])`.

**Reply by thomasmansencal** [reply]
Glad you found out!
